packages/fhir-location-management/src/components/ViewDetails/DetailsTabs/Inventory.tsx

```diff
@@ -266,7 +260,10 @@ export const InventoryView = ({ fhirBaseUrl, locationId }: InventoryViewProps) =
         <span className="d-flex align-items-center">
           <RbacCheck permissions={['Group.update']}>
             <>
-              <Link to={`${baseInventoryPath}/${id}`} className="m-0 p-1">
+              <Link
+                to={`${baseInventoryPath}/${id}`}
+                className={`m-0 p-1 ${id === undefined ? 'disabled-link' : ''}`}
```

Comment:
Could be wrong but is it possible to have an undefined id at this point? we can assume all resources from the server will have an id

Reply:
Noted